peer_mentoring/groups/models.py

```diff
@@ -33,7 +33,9 @@ class Group(models.Model):
     moderator = models.ForeignKey(
         user, on_delete=models.CASCADE, related_name="moderator"
     )
-    banned_users = models.ManyToManyField(User, related_name="banned_from_groups")
+    banned_users = models.ManyToManyField(
+        User, related_name="banned_from_groups", null=True, blank=True
```

Comment:
reviewed previous PR, warning about null

Reply:
What was the warning? Only reason why I have it there is because it won't throws an error on the template if there are no banned users in the group.

Reply:
<img width="1455" alt="image" src="https://github.com/JAWD-001/peer_mentoring/assets/387927/22cb8369-5b30-43b9-bb26-1f70bc238671">